sdk/cdsclient/deprecated.go

```diff
@@ -0,0 +1,61 @@
+package cdsclient
+
+import (
+	"fmt"
+
+	"github.com/ovh/cds/sdk"
+)
+
+func (c *client) ApplicationPipelinesAttach(projectKey string, appName string, pipelineNames ...string) error {
+	uri := fmt.Sprintf("/project/%s/application/%s/pipeline/attach", projectKey, appName)
+	code, err := c.PostJSON(uri, pipelineNames, nil)
+	if err != nil {
+		return err
+	}
+
+	if code >= 300 {
+		return fmt.Errorf("cds: api error (%d)", code)
+	}
+
+	return nil
+}
+
+func (c *client) ApplicationPipelineTriggerAdd(t *sdk.PipelineTrigger) error {
+	uri := fmt.Sprintf("/project/%s/application/%s/pipeline/%s/trigger", t.SrcProject.Key, t.SrcApplication.Name, t.SrcPipeline.Name)
+
+	if t.SrcEnvironment.Name != "" {
+		uri = fmt.Sprintf("%s?env=%s", uri, t.SrcEnvironment.Name)
+	}
+
+	app := &sdk.Application{}
```

Comment:
why get the application from the response but return nil ?

Reply:
yes, no need (as old sdk). Removed.